spec/presenters/petition_presenter_spec.rb

```diff
@@ -66,4 +66,15 @@
       )
     end
   end
+
+  describe '#other_details' do
+    context 'when there is an asking order' do
+      it { expect(subject.other_details).to eq('details') }
+    end
+
+    context 'when the asking order is nil' do
```

Comment:
The important point here is that the presenter doesn't fail, isn't it? So, shouldn't this spec say explicitly that an error/exception doesn't happen?

Reply:
Yep, I agree. I've added an explicit expectation to check it doesn't raise the exception.